Sources/x10/xla_tensor/ir_dump_util.cpp

```diff
@@ -262,5 +279,48 @@ std::string DumpUtil::ToHlo(absl::Span<const Value> values) {
   return ConsumeValue(xla::util::GetComputationHloText(computation));
 }
 
+std::string DumpUtil::GetGraphChangeLog(absl::Span<const Node* const> roots) {
+  auto post_order = Util::ComputePostOrder(roots);
+  std::unordered_map<const Node*, size_t> roots_ids = GetRootsIds(roots);
+  NodeIdMap id_map = GenerateIdMap(post_order);
+  std::vector<ChangeLogNode> change_log;
+  xla::hash_t h = 0x85ebca77c2b2ae63;
```

Comment:
Does this need to stay the same as in: x10/xla_client/util.h ?

Reply:
Not really, can be anything, which is why I didn't make an effort to make a header for it.